shared/charts/gpii-flushtokens/templates/cronjob.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  namespace: {{ .Release.Namespace | quote }}
+  name: {{ template "flushtokens.name" . }}
+spec:
+  schedule: "{{ .Values.cronJobSchedule }}"
+  successfulJobsHistoryLimit: 5
+  failedJobsHistoryLimit: 1
+  concurrencyPolicy: Replace
+  startingDeadlineSeconds: 200
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: {{ template "flushtokens.name" . }}
+        spec:
+          shareProcessNamespace: true
+          containers:
+          - name: gpii-flushtokens
+            image: "{{ .Values.image.repository }}@{{ .Values.image.checksum }}"
+            command: [ "/bin/sh", "-c", "/app/scripts/dockerDeleteExpiredAccessTokens.sh" ]
+            env:
+            - name: GPII_COUCHDB_URL
+              value: '{{ .Values.couchdbUrl }}'
+            # Use '--deleteAll' to delete all of the access tokens
+            - name: DELETE_ALL
+              value: 'false'
+          - name: istio-proxy-manager
+            image: "{{ .Values.image.repository }}@{{ .Values.image.checksum }}"
+            securityContext:
+              allowPrivilegeEscalation: false
+              runAsUser: 1337
+            command: [ "/bin/sh", "-c" ]
+            args:
+              - while ! ps -o user | grep node > /dev/null;
```

Comment:
This is fine but JFYI: `grep -q` :)

Reply:
I learn something new everyday, thanks.  But, based on "don't fix it if it ain't broke", I'll leave it as is.